machine-learning/Machine-Learning-Quiz.md

```diff
@@ -0,0 +1,114 @@
+I've translated the machine learning questions and answers into French:

```

Comment:
can you please change file name from `Machine-Learning-Quiz.md` to `machine-learning-quiz-fr.md`, thanks